src/no/runsafe/accounts/repositories/AccountRepository.java

```diff
@@ -1,31 +1,35 @@
 package no.runsafe.accounts.repositories;
 
-import no.runsafe.framework.api.database.IDatabase;
 import no.runsafe.framework.api.database.ISchemaUpdate;
 import no.runsafe.framework.api.database.Repository;
 import no.runsafe.framework.api.database.SchemaUpdate;
+import no.runsafe.framework.api.player.IPlayer;
+
+import javax.annotation.Nonnull;
 
 public class AccountRepository extends Repository
 {
+	@Nonnull
 	@Override
 	public String getTableName()
 	{
 		return "runsafe_accounts_tokens";
 	}
 
-	public void update(String playerName, String token)
+	public void update(IPlayer player, String token)
 	{
 		this.database.execute(
-			"INSERT INTO `runsafe_account_tokens` (playerName, token) VALUES(?, ?) " +
+			"INSERT INTO `runsafe_account_tokens` (playerName, token, playerID) VALUES(?, ?, ?) " +
 				"ON DUPLICATE KEY UPDATE token = ?",
```

Comment:
This `?` ought be replaced by `VALUES(token)` and the fourth parameter removed, I suppose